h5py/_hl/selections.py

```diff
@@ -91,6 +91,59 @@ def select(shape, args, dsid):
     return sel
 
 
+class MultiBlockSlice(object):
+
+    """
+        A conceptual extension of the built-in slice object to allow selections
+        using start, stride, count and block.
+
+        If given, these parameters will be passed directly to
+        H5Sselect_hyperslab. The defaults are start=0, stride=1, block=1,
+        count=length, which will select the full extent.
```

Comment:
Please have the parameter description here match that of the surrounding classes